packages/contracts-bedrock/deploy/000-L2OutputOracle.deploy.ts

```diff
@@ -10,7 +10,7 @@ const deployFn: DeployFunction = async (hre) => {
   const { deployer } = await hre.getNamedAccounts()
   const { deployConfig } = hre
 
-  const l1 = hre.ethers.getDefaultProvider()
```

Comment:
I thought this would return a provider that is associated with the `--network` config? I guess not

Reply:
Unfortunately not :(